libs/auth/src/card_reader.ts

```diff
@@ -97,6 +100,15 @@ export class CardReader {
     });
   }
 
+  /**
+   * Disconnects the currently connected card, if any
```

Comment:
"connected card" or "connected card reader"?

Reply:
Updated the naming scheme such that it's clearer at both the `CardReader` and `JavaCard` levels that we're talking about disconnecting the card and not the card reader:
- `CardReader.disconnectCard`
- `JavaCard.disconnect`